src/components/Formsfields.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+
+// Define props type for FormField component
+
+interface FormFieldProps {
+    imageSrc: string;
+    placeholderText: string;
+  }
+
+  // FormField component as a functional component
+
+const FormField: React.FC<FormFieldProps> = ({ imageSrc, placeholderText }) => {
+    return (
+      <div className="mt-5 mr-4">
+        <div className="border-2 pl-3 rounded-md py-2 flex gap-4">
+          <Image src={imageSrc} alt="flight" height={200} width={200} className="w-[16px] h-[16px]" />
```

Comment:
There's a confusion here in width and height. You're setting width and height 200px but in styling it's set to 16px. It's getting overridden with 16px but it gets missed for any reason there'll be a huge icon set to 200px.

Kindly remove the classes `w-[16px] h-[16px]` and set the width and height to 16